### ISSUE TITLE
Multiple issues on enabling scrollable plot area for Highstock

### ISSUE THREAD

**Issue by AntonyJos** [highcharts/highcharts#11560]
#### Expected behaviour
On enabling the scrollable plot area for HighStock,
The plot background color should run across the scroll able area.
X Axis should be positioned correctly,
The Navigator.top should be corrected, so that it should be placed below the X Axis
The tooltip header which displays the time should be visible

#### Actual behaviour
On enabling the scrollable plot area for HighStock,
The plot background color is being clipped in between
X Axis goes above the scrollbar.,
The Navigator is placed inside the chart rather than outside
The tooltip header which displays the time is placed behind the X-Axis

#### Live demo with steps to reproduce

1.Create Multiple Y Axises that are arranged one after the other.
2. Add CSS to the chart plot background
```
.highcharts-plot-background {
  fill: black !important;
}
```
3. Enabled scroll able plot Area for the chart 
```
    scrollablePlotArea: {
    	minHeight: 700,
      scrollPositionY: 1,
      opacity: 1
    }
```
4. Have set the height of the div wrapping the container as 500px
```
.chart_container {
  height: 500px;
  overflow: hidden;
}
```
5. Observe the chart being presented

Please find the below working fiddle
https://jsfiddle.net/01gjvtea/

**Observation**

_The plot background color is clipped in middle._

![plot background clipped](https://user-images.githubusercontent.com/42953694/62473531-8dab6500-b7be-11e9-8f56-f88ec7aa1e7c.PNG)

_Navigator is placed inside the plot area_

![Navigator misplaced](https://user-images.githubusercontent.com/42953694/62473632-cb0ff280-b7be-11e9-80fb-9d5bfe2dc4f8.PNG)

_X Axis overlaps the scrollbars_

![X Axis overlapping](https://user-images.githubusercontent.com/42953694/62473729-01e60880-b7bf-11e9-9507-95d89dfb6305.PNG)

_ToolTip Behind the X Axis_

![Tooltip overlapped](https://user-images.githubusercontent.com/42953694/62473902-5c7f6480-b7bf-11e9-9eba-45ace80ba32f.PNG)

#### Product version

v7.1.2

#### Affected browser(s)
Tested out in Google Chrome - ^74.0.x

**Comment by AntonyJos** [comment]
Is there an option to style the scrollbar that is being presented.

Please let me know the impact of setting 

> chart.height
>  chart.scrollablePlotArea.minHeight 

to a stockChart

@pawelfus / @TorsteinHonsi , Kindly look into this issue

I believe the issue with the plot area background has been fixed through  [#10953](https://github.com/highcharts/highcharts/issues/10952) by @TorsteinHonsi . Please verify

**Comment by pawelfus** [comment]
Thanks for reporting all of the issues!

> Please let me know the impact of setting (...) scrollablePlotArea to a stockChart. 

AFAIK `scrollablePlotArea` is not supported in Highstock (not listed in [API](https://api.highcharts.com/highstock/chart)). I'm not sure if this is missing in API, or not supported/tested with Highstock. @TorsteinHonsi ?

> I believe the issue with the plot area background has been fixed through #10953

That's correct, see: https://jsfiddle.net/BlackLabel/o42anq6k/

> X Axis overlaps the scrollbars

Please set `marginRight`: https://jsfiddle.net/BlackLabel/o42anq6k/1/

> Tooltip behind xAxis:

Reported as a separate issue: #11617

**Comment by AntonyJos** [comment]
Any Update on this...?

**Comment by AntonyJos** [comment]
Is there any workaround to get these resolved. Can someone help me out here..

**Comment by TorsteinHonsi** [comment]
This is four issues in one, so I'll split it to make it easier to keep track:

* Plot area, fixed at #10953.
* Tooltip behind axis, reported at #11617.
* Navigator position, see #11670
* Overlap right, see #11671
